lib/game.rb

```diff
@@ -22,6 +26,11 @@ def take_turn
     @current_player, @other_player = @other_player, @current_player
   end
 
+  def exit_game
```

Comment:
Related to the `trap` comment – I am wondering if the `Game` needs to know **anything** about exiting. What do you think?

Reply:
I think the game needs to know if it should keep playing or not, but not really anything about how to exit.  I've taken away it's ability to exit the game, but passed it a piece of state to get information from.